rr/config/app_validator.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python3
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+
+
+class AppValidatortError(RuntimeError):
+    pass
+
+
+class AppValidator():
+
+    def validate(self, cfg):
+        self.validate_streams(cfg)
+        self.validate_filters(cfg)
+        self.validate_actions(cfg)
+        self.validate_triggers(cfg)
+
+    def validate_streams(self, cfg):
```

Comment:
Other than `validate` all the functions in this class don't require `self`, move them to functions